libraries/google_kms_crypto_key.rb

```diff
@@ -66,7 +66,7 @@ def to_s
   end
 
   def primary_state
-    @fetched['primary']['state']
+    @fetched['primary']['state'] unless @fetched.dig('primary', 'state').nil?
```

Comment:
I guess you want to do this 
```suggestion
    @fetched.dig('primary', 'state')
```

Reply:
I don't believe this does the same - I need to return the value of @fetched['primary']['state'], or return nil if any of the values in that nested hash are nil ( avoiding catching NoMethodError if for example @fetched[;primary'] is nil)

Reply:
yeah it does exactly what you have asked for .. reference -> https://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig

Reply:
Attached the behaviour here
<img width="413" alt="Screenshot 2021-11-22 at 11 21 23 PM" src="https://user-images.githubusercontent.com/80091550/142910863-81c0ddae-cb16-468e-87aa-fd200d2e29e9.png">